Workbooks/Network Insights/ApplicationGatewayWebApplicationFirewallPolicies/AppGwWafDefault/AppGwWafDefault.workbook

```diff
@@ -4,9 +4,29 @@
     {
       "type": 1,
       "content": {
-        "json": "## Application Gateway WAF triage workbook\n---\n\nThis workbook will help you triage Application Gateway WAF violations."
+        "json": "WAF data is shown only if Diagnostic Settings are enabled for the Application Gateway associations.\r\n",
+        "style": "info"
       },
-      "name": "text - 2"
+      "name": "text - 5"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "paragraph",
+        "links": [
+          {
+            "id": "feaeca15-1eba-4aa6-bdc5-d948c7f85472",
+            "cellValue": "https://review.learn.microsoft.com/en-us/azure/web-application-firewall/ag/waf-insights",

```

Comment:
Remove the /en-us/ so link defaults to user's preferred language. Also consider using a fw link